domain/domain_test.go

```diff
@@ -19,6 +19,9 @@ import (
 	"crypto/tls"
 	"encoding/json"
 	"fmt"
+	"github.com/pingcap/tidb/parser/mysql"
```

Comment:
Please use ```goimports``` to format it.

Reply:
run ```make bazel_prepare``` to update bazel config.

Reply:
thx @hawkingrei. not sure if it's my goland setting, goimports didn't format the file the way CI wanted. manually fixed.